tor/controller.go

```diff
@@ -133,12 +133,12 @@ func (c *Controller) Start() error {
 }
 
 // Stop closes the connection between the controller and the Tor server.
-func (c *Controller) Stop() error {
+func (c *Controller) Stop() {
 	if !atomic.CompareAndSwapInt32(&c.stopped, 0, 1) {
-		return nil
+		return
 	}
 
-	return c.conn.Close()
+	c.conn.Close()
```

Comment:
Any reason to drop the error here?

Reply:
linter complained, couldn't use `defer torController.Stop()`

Reply:
Perhaps can just make a function closure instead that uses / logs the error?